README.md

```diff
@@ -22,152 +22,7 @@ Swarm
 Swarm is a 2D programming and resource gathering game. Program your
 robots to explore the world and collect resources, which in turn
 allows you to build upgraded robots that can run more interesting and
-complex programs.  Check out the [installation
-instructions](https://github.com/swarm-game/swarm#installing) below,
-join the [IRC channel](COMMUNITY.md), take a look at the
-[wiki](https://github.com/swarm-game/swarm/wiki), or [see how you can
-contribute](CONTRIBUTING.md)!
+complex programs.  More info can be found on the [Swarm
+website](https://swarm-game.github.io).
 
-![World 0 after scanning a tree and making a log.](images/tutorial/log.png)
-
-Features include:
-
-* Practically infinite 2D procedurally generated worlds
-* Simple yet powerful programming language based on the polymorphic
-  lambda calculus + recursion, with a command monad for describing
-  first-class imperative actions
-* Editor support with LSP and highlighting
-* In-game tutorial
-* Multiple game modes:
-    - In Classic mode, you start with the ability to produce only very
-      basic, limited robots; collecting resources allows you to
-      bootstrap your way into programming more sophisticated robots
-      that can explore more of the world, collect more resources, etc.
-    - Creative mode places no restrictions: program robots to your
-      heart's content using whatever language features you want,
-      without worrying about collecting resources.
-    - There are also challenge scenarios where you attempt to program
-      robots in order to solve pre-designed puzzles or challenges.
-
-Installing
-==========
-
-**NOTE**: Compiling Swarm with GHC 9.2.5 and optimizations enabled
-seems to result in very long freezes/delays (tens of seconds) when
-starting Swarm (see
-[#1000](https://github.com/swarm-game/swarm/issues/1000)).  We
-recommend either building Swarm with a different version of GHC
-(*e.g.* 9.4.x), or building with optimizations turned off (which does
-not seem to affect the game performance very much).
-
-**NOTE**: Swarm requires a POSIX-style terminal environment that
-supports `terminfo`.  Linux and MacOS should work out of the box.  On
-Windows, you will need to use [Windows Subsystem for
-Linux](https://learn.microsoft.com/en-us/windows/wsl/); you should
-then be able to follow instructions for installing on Linux.
```

Comment:
Specifically these two notes make more sense to me in swarm repo rather than the website repo.

I can’t imagine going to a website to check if there are any notes for compiling with specific GHC versions.

Reply:
Yes, I agree those notes feel like maybe they don't belong on the website.  How about I put all the instructions for building from source in the repo, and leave the other stuff on the website (with appropriate links)?  Let me try that and see how it looks.

Reply:
@xsebek What do you think now?

Note, I simply removed the note about GHC 9.2.  By now it's pretty old, and since we're recommending building with `stack` (which will now get GHC 9.4) no one would run into this issue unless they (1) want to use their own custom workflow for building via `cabal` or whatever, AND (2) are still using GHC 9.2, which seems like a very unlikely combination.